sbt

```diff
@@ -6,24 +6,24 @@
 
 set -o pipefail
 
-declare -r sbt_release_version="0.13.16"
-declare -r sbt_unreleased_version="0.13.16"
+declare -r sbt_release_version="1.3.2"
+declare -r sbt_unreleased_version="1.3.2"
 
-declare -r latest_213="2.13.0-M2"
-declare -r latest_212="2.12.4"
+declare -r latest_213="2.13.1"
+declare -r latest_212="2.12.10"
```

Comment:
ha, great!

Reply:
this comes with upgrading the `sbt-extras` script